cmd/preflight/cmd/check_operator.go

```diff
@@ -93,7 +93,12 @@ func checkOperatorRunE(cmd *cobra.Command, args []string) error {
 
 	opts := generateOperatorCheckOptions(cfg)
 
-	checkoperator := operator.NewCheck(operatorImage, cfg.Kubeconfig, cfg.IndexImage, opts...)
+	kubeconfig, err := os.Open(cfg.Kubeconfig)
+	if err != nil {
+		return fmt.Errorf("unable to open provided kubeconfig file: %s", err)
+	}
+	defer kubeconfig.Close()
```

Comment:
Do we really want to defer closing this for the entire run? Kind of a nit. Up to you.
Kind of goes with the point below about just passing the byte slice instead of an io.Reader.

Reply:
This open is unrelated - the CLI invocation _always_ has to open a file. That said, might as well close it as soon as we're done with it. I've removed the `defer` and closed it explicitly.